build/tools/wxwidgets.iss

```diff
@@ -51,7 +51,7 @@ Filename: "{app}\wx.url"; Section: "InternetShortcut"; Key: "URL"; String: "http
 
 [Icons]
 Name: "{group}\{cm:ProgramOnTheWeb,wxWidgets}"; Filename: "{app}\wx.url"
-Name: {group}\wxWidgets Manual; Filename: {app}\docs\htmlhelp\wx.chm; WorkingDir: {app}; IconIndex: 0; Flags: useapppaths
+Name: {group}\wxWidgets Manual; Filename: {app}\docs\htmlhelp\wxWidgets-{#WXW_VER}-docs.chm; WorkingDir: {app}; IconIndex: 0; Flags: useapppaths

```

Comment:
I think `wx.chm` is good enough because it's located inside a directory that would typically already contain the version number. Also, `-docs` part is really superficial for a `.chm` file.

I.e. I'd keep the old name, even though I don't have any strong feelings one way or the other.